source/nodejs/adaptivecards-designer/src/containers/berlin/berlin-container.ts

```diff
@@ -6,82 +6,84 @@ import * as hostConfigLight from "../../hostConfigs/berlin-light.json";
 import * as hostConfigDark from "../../hostConfigs/berlin-dark.json";
 
 enum ContainerSize {
```

Comment:
I assume you don't have code change on this file and just format with 4 spaces instead of 8 spaces

Reply:
@licanhua There were a couple of changes - sorry about the line spacing!

I updated lines 45-54 by:

- adding a className to `headerText`
- making `moreButton` a div instead of a button (lines 50-54)